pkg/log/text_formatter.go

```diff
@@ -15,191 +15,67 @@
 package log
 
 import (
+	"bufio"
 	"bytes"
-	"fmt"
-	"runtime"
-	"sort"
 	"strings"
-	"time"
 
 	"github.com/Sirupsen/logrus"
 )
 
-const (
-	nocolor = 0
-	red     = 31
-	green   = 32
-	yellow  = 33
-	blue    = 34
-	gray    = 37
-)
-
-var (
-	baseTimestamp time.Time
-	isTerminal    bool
-)
-
-func init() {
-	baseTimestamp = time.Now()
-	isTerminal = logrus.IsTerminal()
```

Comment:
I have been looking at our various log files and the output from vic-machine. All appear to be consistent, and I don't see the '\n' characters that we used to see before.